### PR TITLE
First draft of specification review period announcment

### PR DESCRIPTION
**Pull request by b-wils**
Includes fix for news URL in header

### CODE REVIEW

File: content/news/lottie-1.0-draft-review/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+date: "2024-08-01T12:01:01.284Z"
+description: "The Lottie Animation Community (LAC), a Joint Development Foundation project, is pleased to announce the public review draft of the v1.0 Lottie JSON specification."
+featuredImage: lac.png
+---
+
+### Lottie Animation Community announces public review draft of Lottie v1.0 specification
+
+The Lottie Animation Community (LAC), a Joint Development Foundation project, is pleased to announce the public review draft of the v1.0 Lottie JSON specification. v1.0 will be the first major step in securing the [Lottie File Format](https://lottie.github.io/) as an open standard for efficient, scalable and cross-platform animated vector graphics.
+
+The public review period for the Lottie v1.0 draft will be from July 29 to August 26, 2024. We request all significant changes be proposed by Aug 19 to allow for final review and publishing. Review the documentation [here](https://lottie.github.io/lottie-spec/) and validate your existing Lotties [here](https://lottie.github.io/lottie-spec/validator). Participate in the review by opening github issues for the spec [here](https://github.com/lottie/lottie-spec/issues).
```
Comment (mbasaglia) [inline content/news/lottie-1.0-draft-review/index.md:12]: Are the dates confirmed?

File: content/news/lottie-1.0-draft-review/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+#### About Lottie v1.0
+
+Version 1.0 of the Lottie Animation Format is the result of the collaborative work of many Lottie stakeholders, since the forming of the Lottie Animation Community and working group. The goal of v1.0 is to include all well known, commonly implemented, documentable features, to establish a baseline for the format and pave the way for future iterations. We envision implementers may want to declare a profile referencing the spec version, to allow for some variance in feature support while driving API consistency and schema reliability.
```
Comment (mbasaglia) [inline content/news/lottie-1.0-draft-review/index.md:16]: Perhaps we should mention that some features that are not widely supported will not be in 1.0 but should be introduced later

File: content/news/lottie-1.0-draft-review/index.md (hunk 1 above)
Comment (b-wils) [reply content/news/lottie-1.0-draft-review/index.md:12]: The date is flexible, we can fill it in once we get everything else ready.

File: content/news/lottie-1.0-draft-review/index.md (hunk 2 above)
Comment (b-wils) [reply content/news/lottie-1.0-draft-review/index.md]: That makes sense, added the following-

"Features that are not widely supported will not be in the 1.0 specification but will be added in a later revision."

**Comment by Aidosmf** [comment]
ok, closing https://github.com/lottie/lottie.github.io/pull/3 :_

**Review by mbasaglia — APPROVED** [review]

**Review by Aidosmf — APPROVED** [review]

**Review by fmalita — APPROVED** [review]

**Pull request merged**